delivery_carrier_package_measure_required/models/stock_quant_package.py

```diff
@@ -13,11 +13,17 @@ class StockQuantPackage(models.Model):
     height_required = fields.Boolean(related="packaging_id.package_height_required")
     weight_required = fields.Boolean(related="packaging_id.package_weight_required")
 
-    @api.constrains("length", "width", "height", "weight")
     # The boolean field use to check if a dimension is required are intentionally left out.
     # To not raise error when changing packaging configuration.
+    @api.constrains("length", "width", "height", "weight")
```

Comment:
No need for quant_ids here for triggering it when the package is filled in ?

Reply:
Sounds like a good idea. added.